projects/table/src/lib/interfaces/show-header-requirements.ts

```diff
@@ -1,4 +1,14 @@
+import { RequireAtLeastOne } from './internal-helpers';
+
 export interface NgxTableShowHeaderRequirements {
 	isLoading: boolean;
 	isEmpty: boolean;
 }
+
+interface NgxTableCypressDataTagsBase {
```

Comment:
Is er een reden waarom je deze niet export?

Reply:
Yes! 

De setup is er omdat je anders een leeg object kan meegeven en dat is wat pointless. Dus daarom staat die RequireAtLeastOne rond deze Base interface zit, zodat je altijd minstens 1 van de 3 tags moet meegeven. 

Dus die base moet niet naar buiten; de NgxTableCypressDataTag die dus minstens 1 van de 3 vereist wel.